src/domains/vote/components/AddressTable/AddressTable.tsx

```diff
@@ -25,47 +31,75 @@ export const AddressTable = ({ wallets, onSelect }: AddressTableProps) => {
 		},
 		{
 			Header: t("COMMON.BALANCE"),
-			accessor: "balance",
-			className: "justify-end",
```

Comment:
Balances should always be aligned to the right afaik

Reply:
Fixed @dated